pkg/monitoring/rules/recordingrules/operator.go

```diff
@@ -0,0 +1,57 @@
+package recordingrules
+
+import (
+	"github.com/machadovilaca/operator-observability/pkg/operatormetrics"
+	"github.com/machadovilaca/operator-observability/pkg/operatorrules"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+const (
+	CommonTemplatesRestoredIncreaseQuery   = "sum(increase(kubevirt_ssp_common_templates_restored_total{pod=~'ssp-operator.*'}[1h]))"
+	TemplateValidatorRejectedIncreaseQuery = "sum(increase(kubevirt_ssp_template_validator_rejected_total{pod=~'virt-template-validator.*'}[1h]))"
+)
+
+func operatorRecordingRules() []operatorrules.RecordingRule {
+	return []operatorrules.RecordingRule{
+		{
+			MetricsOpts: operatormetrics.MetricOpts{
+				Name: "kubevirt_ssp_operator_up",
+				Help: "The total number of running ssp-operator pods",
+			},
+			MetricType: operatormetrics.GaugeType,
+			Expr:       intstr.FromString("sum(up{pod=~'ssp-operator.*'}) OR on() vector(0)"),
```

Comment:
Makes sense to have consts for these queries too? Or are the consts above used somewhere else?

Reply:
they are used in this file: https://github.com/kubevirt/ssp-operator/blob/main/tests/monitoring_test.go#L117 like they were before this refactor

Reply:
FYI: I did used const in other operator I implemented and Joao asked to leave it inline since it's more clear that way.